src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -94,7 +94,7 @@ std::vector<edm4eic::TrackParameters*> eicrecon::TrackSeeding::makeTrackParams(S
       if(theta < 0)
 	{ theta += M_PI; }
       float eta = -log(tan(theta/2.));
-      float pt = 0.3 * R * (m_cfg.m_bFieldInZ * 1000) / 100.;
+      float pt = 0.3 * (R / 1000.) * (m_cfg.m_bFieldInZ * 1000.);
```

Comment:
e.g.
```suggestion
      float pt = 0.3 * (R / dd4hep::mm) * (m_cfg.m_bFieldInZ * Acts::UnitConstants::mm);
```

Reply:
which is wrong. just an example.

Reply:
Ok, I can do that. But I'm a bit confused about the position being used in the simulation now. If I check the units in dd4hep, I see the following:

```
root -l
root [0] #include <Evaluator/DD4hepUnits.h>
root [1] dd4hep::cm
(const double) 1.0000000
root [2] dd4hep::mm
(const double) 0.10000000
```

This has cm set to 1. However, in the standard DD4HEP and EICRecon output ROOT files, unless I'm misreading the data, it looks like the position units for the tracker hits are given in mm. So, how would I use the above to convert from mm to m?

Reply:
I can of course change the 0.3 factor.

Reply:
Unfortunately almost each software has to define its own units header. For now (thanks Wouter):

EDM4hep: GeV, mm, ns (also juggler units)
DD4hep: GeV, cm, s
ROOT: GeV, cm, s
Acts: GeV, mm, mm
Geant4: MeV, mm, ns

So as you see, EDM4hep i.e. our data model uses mm and DD4Hep as well as ROOT geometry uses cm.